Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -92,7 +100,7 @@ public function normalize($object, $format = null, array $context = array())
                 '@context' => $this->contextBuilder->getResourceContext($resource, $context),
                 '@id' => $context['request_uri'],
             ];
-            list($parts, $parameters) = $this->parseRequestUri($context['request_uri']);
+            $uri = $this->parseRequestUri($context['request_uri']);
```

Comment:
What is the exact problem with my implem? I'm not against using the league package but I'll strongly prefer avoiding an external dependency here.